README.md

```diff
@@ -1,25 +1,13 @@
-# hubot
+# hubot-for-hubot
 
-hubot is a chat bot built on the [Hubot][hubot] framework. It was
-initially generated by [generator-hubot][generator-hubot], and configured to be
-deployed on [Heroku][heroku] to get you up and running as quick as possible.
-
-This README is intended to help get you started. Definitely update and improve
-to talk about your own instance, how to use and deploy, what functionality is
-available, etc!
-
-[heroku]: http://www.heroku.com
-[hubot]: http://hubot.github.com
-[generator-hubot]: https://github.com/github/generator-hubot
+hubot is a chat bot built on the [Hubot][hubot] framework for helping the [Hubot community on Slack](https://hubot-slackin.herokuapp.com/). 
```

Comment:
the `[hubot]` link isn't defined.  Also there are a lot of `hubot`s in that sentence. 

> This is an instance of [Hubot][hubot] for helping the [Hubot community on Slack](https://hubot-slackin.herokuapp.com/).